massa-final-state/Cargo.toml

```diff
@@ -9,17 +9,18 @@ displaydoc = "0.2"
 nom = "7.1"
 thiserror = "1.0"
 # custom modules
-massa_ledger = { path = "../massa-ledger" }
+massa_ledger_exports = { path = "../massa-ledger-exports" }
+massa_ledger_worker = { path = "../massa-ledger-worker" }
```

Comment:
I don't understand what you mean

Reply:
FI: it means having a controller trait that wraps the disk ledger db module so that other crates only need to import the trait and have no RocksDB dependency

Reply:
Will add this on Monday